Tests/validate_schema.py

```diff
@@ -1,14 +1,22 @@
 import sys
 from pykwalify.core import Core
 
-
 def main(argv):
     if len(argv) < 2:
         print('you must provide file path & content-type')
         sys.exit(1)
 
     file_path = argv[0]
     schema_path = argv[1]
+    
+    is_local = False
+    if len(argv) == 3:
+        is_local = argv[2]
+
+    if is_local:
```

Comment:
note that if you pass `"False"`  string to this - it will go inside the if..
in python
```
if "False":
     print "I'm in"
```
will print "I'm in"